### ISSUE TITLE
[Contacts] In the 'Create New Contact' form, if the Disease is not selected before choosing the Source Case, the form cannot be saved

### ISSUE THREAD

**Issue by roxanamlendea** [hzi-braunschweig/SORMAS-Project#7400]
### Bug Description
When creating a New Contact, if the Disease is not selected before choosing the Source Case, the form cannot be saved. It can only be saved if the Disease is selected  before choosing the Source Case.

### Steps to Reproduce

1. Go to the Contacts page.
2. Open the New Contact form.
3. Fill in the name fields and select the sex, but leave the Disease field empty.
4. Choose a Source Case.
5. Click on Save.

### Actual Behavior
The form cannot be saved and a message stating that the user needs to specify a valid disease appears.

### Expected Behavior
The form should be saved and created.

### Screenshots

![new contact](https://user-images.githubusercontent.com/89522688/144204643-3b11c2ce-c3f9-4bd2-ad0e-93c520f6d9c9.gif)

### System Details

* Device: Windows 10
* SORMAS version: 1.66.0-SNAPSHOT
* Android version/Browser: Chrome
* Server URL: test-de.sormas.netzlink.com
* User Role: NatUser

**Comment by kwa20** [comment]
@roxanamlendea I think the diseases might not be reduced to only one disease for that instance?

**Comment by roxanamlendea** [comment]
> @roxanamlendea I think the diseases might not be reduced to only one disease for that instance?

Yes, it's not reduced to only one disease, the instance has multiple diseases.

**Comment by roxanamlendea** [comment]
Verified ticket on local environment using the latest version of Sormas 1.67.0-SNAPSHOT from the development branch.
